ddh-intro.md

```diff
@@ -2,28 +2,19 @@
 
 ![](http://docs.duckduckhack.com/assets/hack_search_engine.png)
 
-**We're a community dedicated to making DuckDuckGo the best search engine for programmers, and you can help!**
+**We're an open-source community dedicated to improving DuckDuckGo search results with Instant Answers, and you can help!**
 
-Our goal is to have a high quality, relevant Instant Answer for every search.
+Our goal is to have a high quality, relevant Instant Answer for every search. 
 
-Instant Answers are built by leveraging 3rd party APIs and external data sets. The community and DuckDuckGo staff work with developers to make sure the Instant Answer is the best it can be.
-
-## Programming Mission
```

Comment:
Let's change this to "Getting Started" to match the repos and provide a clear CTA.
We can then can remove the "See how to get started ..." line below that.